src/entities/elements/touchable_elements/block.py

```diff
@@ -0,0 +1,20 @@
+from src.entities.elements.element import Element
+from src.enums.element_types.block_sub_type import BlockSubTypeEnum
+from src.enums.element_types.element_type import ImageConfigTypeEnum
+from src.utils.image_mappings import image_configurations
+from src.utils.position import Position
+from src.utils.size import Size
+
+
+class Block(Element):
+    def __init__(
+        self,
+        x: int,
+        y: int,
+        type: str = BlockSubTypeEnum.OVERWORLD_BLOCK.value,
```

Comment:
Instead of passing strings to identify subelements, it's better to use an enum. This restricts the types of subelements we can create and avoids issues with invalid strings.

_ = Block(12,12,"unexisting-block")
_ = Block(1,12,BlockSubTypeEnum.OVERWORLD_BLOCK)

Reply:
Yes you can made it using this form '_ = Block(1,12,BlockSubTypeEnum.OVERWORLD_BLOCK.value)'

Reply:
but if we have Enums we should use them , and also if we receive a string we can have posible bugs in a future, I think i better if we restrict the possible arguments the constructor can have

Reply:
But if you see the others enums we are managing using strings, getting the values of enums which are constants so I based on it

Reply:
Yes you are correct, I'm going to change them

Reply:
@daniel-espinoza-79  Done